docs/pages/enroll-resources/server-access/guides/host-user-creation.mdx

```diff
@@ -261,6 +261,13 @@ the potential impacts.
 Should a Teleport SSH instance be restarted while a session is in progress, the user
 will be cleaned up at the next Teleport restart.
 
+## Migrating unmanaged users
+
+Host users created with `create_host_user_mode: keep` prior to `v14.3.24`, `v15.4.16`, and `v16.1.8`
```

Comment:
I don't think we've cut a new release for `v14.3.24` yet, but I'm assuming this will be the next version number?